pkg/controllers/capr/controllers.go

```diff
@@ -30,6 +30,8 @@ func Register(ctx context.Context, clients *wrangler.Context, kubeconfigManager
 		ReleaseData:             capr.GetKDMReleaseData,
 		SystemAgentImage:        settings.SystemAgentInstallerImage.Get,
 		SystemPodLabelSelectors: systeminfo.NewRetriever(clients).GetSystemPodLabelSelectors,
+		PreBootstrapCluster:     prebootstrap.NewRetriever(clients).PreBootstrapCluster,
```

Comment:
why not just simplify this and make it so that if `GetPreBootstrapManifests` returns content, then you know the cluster has to be pre-bootstrapped, vs adding 2 functions